test_scripts/Defects/5_0/2454_SDL_does_not_clear_app_info_after_MASTER_RESET.lua

```diff
@@ -0,0 +1,59 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2454
+--
+-- Reproduction Steps:
+-- HMI and SDL are started.
+-- Register and activate Application1.
+-- Register and activate Application2.
+-- From HMI send notification OnExitAllApplication{reason = MASTER_RESET} to SDL.
+
+-- Expected Behavior:
+-- All SDL data are cleaned and reset.
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require("user_modules/script_runner")
+local common = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local function HMISendToSDL_MASTER_RESET()
+	common.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications",
+		  { reason = "MASTER_RESET" })
+	common.getMobileSession(1):ExpectNotification("OnAppInterfaceUnregistered",
+          { reason = "MASTER_RESET" })
+	common.getMobileSession(2):ExpectNotification("OnAppInterfaceUnregistered",
+          { reason = "MASTER_RESET" })
+		  :ValidIf(function()
+                local app_info_table = utils.jsonFileToTable(commonPreconditions:GetPathToSDL()  .. "/app_info.dat")
+                local resumption_data = app_info_table.resumtion
+        		if resumption_data == nil then
```

Comment:
`resumption` not becomes empty, apps data contains in `resume_app_list`, and also `resume_app_list` becomes not nil but empty table. must be compared table `resume_app_list` with `{}` empty one, but maybe must be removed all data from app_info.dat, need to clarify expectation.

Reply:
@GetmanetsIrina Changed in 8f122c03b766c58324322f3c16e9d9f38df06596.